Source/Platform/DefaultOpenTK/Backend/Graphics/NativeWindow.cs

```diff
@@ -235,21 +267,39 @@ private void OnUserDataChanged(object sender, EventArgs e)
 
 			// Apply the target state to the game window wherever values changed
 			if (this.internalWindow.WindowState != targetWindowState)
+			{
```

Comment:
There is no specific code style rule for this, but I'd argue that the code has become less readable by transforming all the "one liner ifs" into blocks that happen to contain just one line. Please stick to the existing one-liners unless there's a specific reason for changing each one.

(I've marked this one instance, but the same argument goes for all similar changes in this file.)